util/zip-exact/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "mc-util-zip-exact"
+version = "1.3.0-pre0"
+authors = ["MobileCoin"]
+edition = "2018"
```

Comment:
We're moving to 2021, I think you could safely do that here.

Reply:
Are we sure sure about that? We should create some of the tests that test out whether the problem this was designed to solve (i.e. proofs validity being subverted by unequal length collections) is actually solved?

Reply:
I'm motivated to learn the MLSAG math better, so I'll officially volunteer to write that test coverage

Reply:
Are these comments in the right place?